src/pages/index.js

```diff
@@ -58,187 +47,74 @@ export default function IndexPage({ pkgVersion }) {
 
   return (
     <>
-      <Head>
-        <title>CHROMICONS</title>
-        <link
-          rel="shortcut icon"
-          href={`${process.env.BACKEND_URL}/favicon.ico`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="16x16"
-          href={`${process.env.BACKEND_URL}/favicon-16x16.png`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="32x32"
-          href={`${process.env.BACKEND_URL}/favicon-32x32.png`}
-        />
-        <link
-          rel="apple-touch-icon"
-          sizes="180x180"
-          href={`${process.env.BACKEND_URL}/apple-touch-icon.png`}
-        />
-        <link
-          rel="apple-touch-icon-precomposed"
-          type="image/png"
-          sizes="144x144"
-          href={`${process.env.BACKEND_URL}/chroma@2x.png`}
-        />
-        <link
-          rel="apple-touch-icon-precomposed"
-          type="image/png"
-          sizes="114x114"
-          href={`${process.env.BACKEND_URL}/chroma@2x.png`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="192x192"
-          href={`${process.env.BACKEND_URL}/android-icon-192x192.png`}
-        />
-        <link
-          rel="icon"
-          type="image/png"
-          sizes="512x512"
-          href={`${process.env.BACKEND_URL}/android-icon-512x512.png`}
-        />
-        <link
-          rel="manifest"
-          href={`${process.env.BACKEND_URL}/manifest.json`}
-        />
-        <meta name="msapplication-TileColor" content="#ffffff" />
-        <meta
-          name="msapplication-TileImage"
-          content={`${process.env.BACKEND_URL}/chroma@2x.png`}
-        />
-        <meta name="theme-color" content="#ffffff" />
-        <meta
-          name="viewport"
-          content="width=device-width, initial-scale=1.0"
-        ></meta>
-      </Head>
-
       <IconModal icon={iconInView} onDismiss={() => setIconInView(null)} />
 
       <header
-        className="px-4 sm:px-6 lg:px-16 bg-gray-800 pb-32 bg-cover bg-no-repeat bg-center"
+        className="mb-16"
         style={{
-          backgroundImage: `url(${process.env.BACKEND_URL}/bg-cover.svg)`,
+          background: '#fff',
```

Comment:
Instead of an inline style here, can we use a tailwind utility class instead `bg-white` ?